src/Redaxtor.js

```diff
@@ -282,6 +289,22 @@ class Redaxtor {
             this.barNode
         );
         options.navBarRoot.appendChild(this.barNode);
+
+    }
+    /**
+     * Renders tbe hover overlay
+     */
+    showHoverOverlay(root){
+        this.overlayNode = document.createElement("DIV");
```

Comment:
Use something custom here, i.e. `redaxtor-overlay` instead of div
We are currently following idea that all redaxtor stuff should be visible in DOM inspector